src/ensembl/src/shared/state/ens-object/ensObjectActions.ts

```diff
@@ -86,7 +86,7 @@ export const fetchEnsObject = (
     dispatch(
       fetchEnsObjectAsyncActions.success({
         id: ensObjectId,
-        data: response,
+        data: response
```

Comment:
@imransl @azangru it looks like we all have different settings here. Should we have a discussion about keeping it consistent?

Reply:
Prettier seems to be doing the right thing now, following the settings in prettier.config.json. If you ever see commits that contain trailing commas, it would be interesting to investigate why.

There was a brief period when we had different versions of prettier, and the latest version defaulted to placing trailing commas everywhere. I think we are dealing with the legacy of that period now.

Reply:
I suppose this is something we can't avoid now till all trailing commas are erased by prettier. It is annoying that they enabled trailing commas by default. This probably makes me wonder whether we should not rely too much on their defaults and override all of them in our prettier config.